### PR TITLE
Zeckendorf's theorem implementation

### LINKED ISSUE #126: Zeckendorf’s Theorem in C++ and JAVA
To represent a positive number as a sum of distinct, non-neighboring Fibonacci numbers

### PR DESCRIPTION
**Pull request by mayank-kumarr**
close #126 
Zeckendorf's theorem, which states that **Every positive integer can be written uniquely as a sum of distinct non-neighboring Fibonacci Numbers**, is implemented here!

### CODE REVIEW

**Comment by rishabhgarg25699** [comment]
@mayankkumar1205 Add closes #issue  statement

**Comment by mayankkumar1205** [comment]
@rishabhgarg25699 Do I need to close pull request?

**Comment by mayank-kumarr** [comment]
> @mayankkumar1205 Add closes #issue statement

@rishabhgarg25699 Added!!

**Comment by rishabhgarg25699** [comment]
@mayankkumar1205 Same as previous PR

**Comment by mayank-kumarr** [comment]
@rishabhgarg25699 Changed pr number to issue number!

File: Zeckendorf's Theorem.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        int f = fib(n);
+        printf("%d ", f);
+        n = n - f;
+    }
+}
+int main()
+{
+    int n;
+	cout<<"Enter a number: ";
```
Comment (rishabhgarg25699) [inline Zeckendorf's Theorem.cpp:29]: @mayankkumar1205 No proper Indentation.

**Review by rishabhgarg25699 — CHANGES_REQUESTED** [review]

File: Zeckendorf's Theorem.cpp (hunk 1 above)
Comment (mayank-kumarr) [reply Zeckendorf's Theorem.cpp:29]: > @mayankkumar1205 No proper Indentation.

@rishabhgarg25699 Sorry, I didn't see it. Made the indentation properly now!

**Review by rishabhgarg25699 — APPROVED** [review]

**Pull request merged**